x-pack/plugins/security_solution/public/common/components/exceptions/edit_exception_modal/index.tsx

```diff
@@ -146,6 +146,11 @@ export const EditExceptionModal = memo(function EditExceptionModal({
     }
   }, [shouldDisableBulkClose]);
 
+  const isSubmitButtonDisabled = useMemo(
```

Comment:
Ahhh, nice 😉  I was gonna say maybe we need to return the errors from the builder, but really the only errors would be empty values or invalid values that would show with a red error state in the inputs.